frontend/src/pages/accounts/profile.module.scss

```diff
@@ -1,5 +1,6 @@
 @import "../../styles/tokens.scss";
 @import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "~@mozilla-protocol/core/protocol/css/includes/mixins/typography";
```

Comment:
@Vinnl I tried using this to import the Zilla Slab font (this line exists in `Button.module.scss`), but was wondering if its redundant.

Reply:
I think it's technically redundant if `Button.module.scss` gets loaded as well, but since that means removing it from there would also break it here, and also because that means people reading this file can inspect the imports to see where the mixin came from, I'd indeed include this.